### PR TITLE
Create basic Android project template

### LINKED ISSUE #3: Create empty android project
Create a basic Android application project and have one test running.

### PR DESCRIPTION
**Pull request by solidi**
# Problem

No Android project to work on.

# Solution

Setup new project with bare minimum files to have project compiling and test passing.

# Addresses

Closes #3 

# Notice

Lets include .idea files that will sustain code style over the course of the project.

### CODE REVIEW

File: katana/build.gradle [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        applicationId "com.mototdd.katana"
+        minSdkVersion 23
```
Comment (solidi) [inline katana/build.gradle:10]: @exallium - Why 23? Because this is the minimum version for insta apps + android auto.

File: katana/build.gradle [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
+    }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation 'com.android.support:appcompat-v7:27.1.0'
```
Comment (exallium) [inline katana/build.gradle]: consider pulling `27.1.0` into it's own variable.  We will use this version number a lot.

File: katana/build.gradle [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    // https://github.com/mannodermaus/android-junit5
+    testImplementation 'junit:junit:4.12'
```
Comment (exallium) [inline katana/build.gradle]: is this for JUnit5?

File: katana/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mototdd.katana">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
```
Comment (exallium) [inline katana/src/main/AndroidManifest.xml]: Are we supporting RTL?

**Review by exallium — APPROVED** [review]
A couple questions

File: katana/build.gradle (hunk 2 above)
Comment (solidi) [reply katana/build.gradle]: Agreed.
- [x] Create ext version variables.

File: katana/build.gradle (hunk 3 above)
Comment (solidi) [reply katana/build.gradle]: Ah, forgot to investigate this. Here, no, it is still 4.x. I will create a ticket for this upgrade. It does not appear as easy as I thought.

- [x] Create jUnit5 migration ticket.

File: katana/src/main/AndroidManifest.xml (hunk 4 above)
Comment (solidi) [reply katana/src/main/AndroidManifest.xml]: This appears to be an artifact of the project generation and I did not intentionally add it.

Lets discuss this at our next meeting. My experience tells me that the small cost of adding international/localizing the UI may give this us a larger audience plus increased learnings and findings.

- [x] Remove RTL and localization.

**Pull request merged**